crates/prql_compiler/src/semantic/context.rs

```diff
@@ -6,14 +6,17 @@ use std::{collections::HashMap, fmt::Debug};
 use super::*;
 use crate::ast::pl::*;
 use crate::error::Span;
+use crate::utils::IdGenerator;
 
 /// Context of the pipeline.
-#[derive(Default, Serialize, Deserialize, Clone)]
```

Comment:
Context is meant be serializable, so it can be persisted to disk. This is not used right now, but there are plans to do it.